### PR TITLE
Add Finish chapter_convolutional-modern batch-norm

### PR DESCRIPTION
**Pull request by archersama**
*Description of changes:Finish chapter_convolutional-modern batch-norm*

Since there is no discussion of tensorflow, I use '? ', instead of discussion address, can be changed later when there is a discussion address.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_convolutional-modern/batch-norm.md [hunk 1]
```diff
@@ -507,6 +663,13 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 ```
 
+
+
+```{.python .input}
+#@tab tensorflow
+d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
```
Comment (terrytangyuan) [inline chapter_convolutional-modern/batch-norm.md:670]: We haven't defined `train_ch6` for `d2l.tensorflow` yet though. Have you verified that your changes work?

File: chapter_convolutional-modern/batch-norm.md [hunk 2]
```diff
@@ -448,6 +575,12 @@ net[1].gamma.reshape((-1,)), net[1].beta.reshape((-1,))
 
 ```
 
+```{.python .input}
+#@tab tensorflow
+net.get_layer(index=1).gamma,net.get_layer(index=1).beta
```
Comment (terrytangyuan) [inline chapter_convolutional-modern/batch-norm.md:580]: `net.layers[1]` is more consistent with other implementations

File: chapter_convolutional-modern/batch-norm.md (hunk 1 above)
Comment (archersama) [reply chapter_convolutional-modern/batch-norm.md:670]: actually，in the local environment, I defined a train_ch6 in my package。By the way, I noticed that you also used from d2l import tensflow as d2l, but the package about tensorflow has not been used, are you also defining it locally?

File: chapter_convolutional-modern/batch-norm.md (hunk 2 above)
Comment (archersama) [reply chapter_convolutional-modern/batch-norm.md:580]: got it，thank you

File: chapter_convolutional-modern/batch-norm.md (hunk 1 above)
Comment (terrytangyuan) [reply chapter_convolutional-modern/batch-norm.md:670]: It's actually being defined and used in previous chapters. You can find them in d2l/tensorflow.py.

**Comment by terrytangyuan** [comment]
This has been taken over and completed as discussed over Slack. @archersama Thanks for the efforts! We look forward to your contributions after your exams!

**Pull request closed without merging**